compose/django.md

```diff
@@ -62,25 +64,25 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
 
 9. Add the following configuration to the file.
 
-    ```none
-     version: '3'
+    ```yaml
+    version: "{{ site.compose_file_v3 }}"
```

Comment:
Hmm.. this somehow adds a space in front of each line. I recall we had a similar issue at some point; https://github.com/docker/docker.github.io/pull/8105 (also somewhere else, but trying to find what the solution was for that)